api/src/modules/sourcing-record-groups/sourcing-record-group.entity.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  BaseEntity,
+  Column,
+  Entity,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { User } from 'modules/users/user.entity';
+import { BaseServiceResource } from 'types/resource.interface';
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+
+export const sourcingRecordGroupResource: BaseServiceResource = {
+  className: 'SourcingRecordGroup',
+  name: {
+    singular: 'sourcingRecordGroup',
+    plural: 'sourcingRecordGroups',
+  },
+  entitiesAllowedAsIncludes: [],
+};
+
+@Entity()
+export class SourcingRecordGroup extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  @ApiProperty()
+  id!: string;
+
+  @Column({ nullable: true })
+  @ApiProperty()
+  title!: string;
+
+  @Column({ nullable: true })
+  @ApiPropertyOptional()
+  description?: string;
+
+  @Column({ type: 'jsonb', nullable: true })
+  @ApiPropertyOptional()
+  metadata?: JSON;
+
+  @Column({
+    type: 'timestamp',
+    default: () => 'CURRENT_TIMESTAMP',
+  })
+  @ApiProperty()
+  lastEdited!: string;
+
+  @ManyToOne(() => User, (user: User) => user.sourcingRecordGroups, {
+    eager: false,
+  })
+  @ApiProperty()
+  lastEditedUser?: User;
+
+  /**
```

Comment:
Causing API to crash, this time not sure which decorator you want to let here - Either `line 46 `or `line 49`
Take in account that making:

```suggestion
  @ApiProperty({type: ()=> User})
```
Will document that the API will retrieve full user info when retrieving a Sourcing Record. I believe this does not reflect the reality